src/components/seeds/database/index.tsx

```diff
@@ -265,6 +265,27 @@ export default function Database(props: DatabaseProps): JSX.Element {
     }
   }, [query, location, history, setSearchCriteria, selectedOrganization, searchCriteria]);
 
+  useEffect(() => {
+    if (selectedOrganization) {
+      const populateUnfilteredResults = async () => {
+        const apiResponse = await search({
+          prefix: 'facilities.accessions',
+          fields: ['id'],
+          search: convertToSearchNodePayload({}, selectedOrganization.id),
+          count: 1000,
+        });
+
+        setUnfilteredResults(apiResponse);
+      };
+      const populatePendingAccessions = async () => {
+        const data = await getPendingAccessions(selectedOrganization.id);
+        setPendingAccessions(data);
+      };
+      populateUnfilteredResults();
+      populatePendingAccessions();
+    }
+  }, [selectedOrganization]);
```

Comment:
[selectedOrganization.id]?

Reply:
I'm getting the linter "React Hook useEffect has a missing dependency: 'selectedOrganization'. Either include it or remove the dependency array  react-hooks/exhaustive-deps" if I use selectedOrganization.id :(